### PR TITLE
Notification Center - User Preference- start design 

### PR DESCRIPTION
**Pull request by ainouzgali**
Screen Shot 2022-07-27 at 18 46 04

### CODE REVIEW

File: packages/notification-center/src/components/notification-center/components/user-preference/UserPreference.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      { type: 'push', active: false },
+      { type: 'sms', active: false },
+      { type: 'email', active: true },
+    ],
+  },
+];
+
+export function UserPreference({ setShowSettings }: { setShowSettings: (boolean) => void }) {
+  const { theme, common } = useNovuThemeProvider();
+  const baseTheme = theme?.notificationItem?.unseen;
```
[the remark this answers is not in the document]
Comment (ainouzgali) [reply packages/notification-center/src/components/notification-center/components/user-preference/UserPreference.tsx:37]: No, it just happens to be the needed colors. I am opening an issue about it right now. We need to maybe make the user's defined colors more generic somehow. Like maybe just 3 main colors,2 background colors and a font.  Or something like that. We can discuss it @djabarovgeorge

File: packages/notification-center/src/components/notification-center/components/user-preference/ChannelPreference.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      >
+        {channel.icon}
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'flex-start',
+            padding: '0px',
+            gap: '5px',
+          }}
```
Comment (djabarovgeorge) [inline packages/notification-center/src/components/notification-center/components/user-preference/ChannelPreference.tsx:41]: what do you think about changing to styled-components here so there will be less context here and the components will be more informative?

**Review by djabarovgeorge — COMMENTED** [review]
Looks really great, loved the UI screenshot
left you some questions and suggestions.

File: packages/notification-center/src/components/notification-center/components/user-preference/ChannelPreference.tsx (hunk 2 above)
Comment (ainouzgali) [reply packages/notification-center/src/components/notification-center/components/user-preference/ChannelPreference.tsx:41]: Yes! Forgot to get back to that. Thanks!

File: packages/notification-center/src/components/notification-center/components/user-preference/UserPreference.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (djabarovgeorge) [reply packages/notification-center/src/components/notification-center/components/user-preference/UserPreference.tsx:37]: In the initial theme support, there were common colors but that made the configuration more complex to add and maintain so we removed it, we most definitely can raise this one again.

**Pull request merged**